### ISSUE TITLE
[FEATURE] Remove blank comments

### ISSUE THREAD

**Issue by andreyfedoseev** [astral-sh/ruff#4238]
I couldn't find a rule for removing blank comments, so I made one myself.

I can submit a PR if you find that useful, I only need to know what error code to use (I currently use `RUF101` locally)

## What it does
Check for blank comments.

## Why is this bad?
Blank comments are useless and should be removed.

## Example
```python
print("Hello, World!")  #
```

Use instead:
```python
print("Hello, World!")
```

**Comment by charliermarsh** [comment]
I agree that this could be useful. I could see a case for either `RUF010` (the `RUF` rules aren't meaningfully categorized by code, so it's fine to just use the next available), or `ERA002` (since `ERA001` deals with commented-out code). Leaning towards `RUF010` though to avoid deviating from an "upstream" plugin.

**Comment by charliermarsh** [comment]
This was implemented in the latest release as a pylint rule (https://github.com/astral-sh/ruff/pull/9174).
